File: uri/1 - Basics/1065/Main.java
import java.util.*;
class Main{
    public static void main(String [] args){
        Scanner scanner = new Scanner(System.in);
        int par = 0;
        int a = scanner.nextInt();
        if(a%2 == 0){
             par++;
        }
        int b = scanner.nextInt();
        if(b%2 == 0){
             par++;
        }
        int c = scanner.nextInt();
        if(c%2 == 0){
             par++;
        }
        int d = scanner.nextInt();
        if(d%2 == 0){
             par++;
        }
        int e = scanner.nextInt();
        if(e%2 == 0){
             par++;
        }
        
        System.out.printf("%d valores pares\n", par);
    }
}

File: uri/1 - Basics/1065/Main.go
package main

import "fmt"

func main() {
	var a, b, c, d, e int
	par := 0

	fmt.Scanf("%d", &a)
	if a%2 == 0 {
		par++
	}
	fmt.Scanf("%d", &b)
	if b%2 == 0 {
		par++
	}
	fmt.Scanf("%d", &c)
	if c%2 == 0 {
		par++
	}
	fmt.Scanf("%d", &d)
	if d%2 == 0 {
		par++
	}
	fmt.Scanf("%d", &e)
	if e%2 == 0 {
		par++
	}

	fmt.Printf("%d valores pares\n", par)
}
